### ISSUE TITLE
press section techcrunch logo no showing up

### ISSUE THREAD

**Issue by ashu-sharma07** [londonappbrewery/TinDog-Start#35]
In the press section of the webpage the first branc logo of techcruch is not showing up as the src got techcrunch.png but the file name is TechCrunch.png
      <img src="images/techcrunch.png" alt="tc-logo" />

**Comment by monteiroh** [comment]
> In the press section of the webpage the first branc logo of techcruch is not showing up as the src got techcrunch.png but the file name is TechCrunch.png ![tc-logo](images/techcrunch.png)

Yes, you are right, I found the same problem. Two differents solutions are: 1) Rename the image to techcrunch.png
2) Rename the img source to "TechCrunch.png", the new line will be `<img` src="images/TechCrunch.png" alt="tc-logo">`
Warning: Both solutions don't work together, only choose one.

**Comment by ashu-sharma07** [comment]
yeah hope they will fix it I am creating a pull request
